Makefile

```diff
@@ -97,12 +97,11 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
-test: manifests generate fmt vet envtest ## Run tests.
+test: manifests generate fmt vet envtest ## Run unit tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
-# Run e2e tests
 .PHONY: e2e-test
-e2e-test:
+e2e-test: ## Run e2e tests on kind cluster.
```

Comment:
it doesn't necessarily be on kind, we run kind in our CI but you can run this on any cluster

Reply:
Good point updated this.